docs_site/instruments.md

```
@@ -37,47 +33,75 @@ nifty_ce = Option(
+        for i in results:
+            print(i.exchange, i.symbol)
+    ```
 
-with TTConnect("zerodha", config) as broker:
-    expiries = broker.get_expiries(underlying)
-    print("Expiries:", expiries)
+=== "Async"
 
-    futures = broker.get_futures(underlying)
-    print("Futures count:", len(futures))
+    ```python
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)